pulsar-client-tools/src/main/java/org/apache/pulsar/admin/cli/CmdNamespaces.java

```diff
@@ -847,6 +847,26 @@ void run() throws PulsarAdminException {
         }
     }
 
+    @Parameters(commandDescription = "get bundle stats from namespace")
+    private class BundleStats extends CliCommand {
+        @Parameter(description = "tenant/namespace", required = true)
+        private java.util.List<String> params;
+
+        @Parameter(names = { "--bundle", "-b" }, description = "{start-boundary}_{end-boundary}")
```

Comment:
It'd be nice to update the description to mention that omitting the field will get all bundle stats for the namespace. (I see that the other bundle commands don't have this note, but I still think it'd be helpful.)